layers/traffic.py

```diff
@@ -0,0 +1,52 @@
+import csv
+from utils import *
+
+
+def makeDftVehicleCounts():
+    tmp = "tmp_vehicle_counts"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    run(
+        [
+            "wget",
+            # From https://roadtraffic.dft.gov.uk/downloads
+            "https://storage.googleapis.com/dft-statistics/road-traffic/downloads/data-gov-uk/dft_traffic_counts_aadf.zip",
+            "-O",
+            f"{tmp}/dft_traffic_counts_aadf.zip",
+        ]
+    )
+    run(["unzip", f"{tmp}/dft_traffic_counts_aadf.zip", "-d", tmp])
+
+    gj = {
+        "type": "FeatureCollection",
+        "features": [],
+    }
+
+    with open(f"{tmp}/dft_traffic_counts_aadf.csv") as f:
+        for row in csv.DictReader(f):
+            # Only keep the latest year
```

Comment:
Dunno what people will want to see; starting with something simple to visualize.